src/app/(proper_react)/redesign/Shell.tsx

```diff
@@ -33,6 +34,7 @@ export const Shell = (props: Props) => {
   return (
     <>
       <GaScript nonce={props.nonce} />
+      <SubscriptionCheck />
```

Comment:
This change was supposed to already be in `main`. I assume that it might have been removed while resolving merge conflicts or I’ve never pushed it up.